distributed/ddp-tutorial-series/slurm/sbatch_run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#SBATCH --job-name=multinode-example
```

Comment:
Turn these into command line arguments

Reply:
Not a slurm power user but I've only seen these hardcoded in the scripts (https://docs-research-it.berkeley.edu/services/high-performance-computing/user-guide/running-your-jobs/scheduler-examples/)

Reply:
Yeah, that was my understanding with sbatch as well. Not sure if we can make them cmdline args.